step-templates/aws-add-tags-to-ec2-instance.json

```diff
@@ -0,0 +1,79 @@
+    "Octopus.Action.RunOnServer": "false"
+  },
+  "Parameters": [
+    {
+      "Id": "3670adeb-3e4f-4c55-829b-b39bb709d841",
+      "Name": "odInstanceId",
+      "Label": "Instance ID",
+      "HelpText": "The EC2 Instance ID of the Instance you would like to add Tags to.",
+      "DefaultValue": "i-xxxxxxxxxxxxxxxxx",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      },
+      "Links": {}
+    },
+    {
+      "Id": "918acd83-39ae-47d1-9682-12333e723cff",
+      "Name": "odAction",
+      "Label": "Action",
+      "HelpText": "The action you would like to perform - Add or Remove Tags from an EC2 Instance.",
+      "DefaultValue": "New",
+      "DisplaySettings": {
+        "Octopus.ControlType": "Select",
+        "Octopus.SelectOptions": "New|Add tag to an instance\nRemove|Remove tag from an instance"
+      },
+      "Links": {}
+    },
+    {
+      "Id": "9b4d592a-712d-459e-af73-1db757a769d8",
+      "Name": "odTags",
+      "Label": "Tags",
+      "HelpText": "The Tags you would like to Add or Remove from an EC2 Instance, for example:\n\n- Name=MyProject\n- Environment=Prod\n- Version=1.3.3.7",
+      "DefaultValue": "Name=#{Octopus.Project.Name}\nEnvironment=#{Octopus.Environment.Name}\nVersion=#{Octopus.Release.Number}",
+      "DisplaySettings": {
+        "Octopus.ControlType": "MultiLineText"
+      },
+      "Links": {}
+    },
+    {
+      "Id": "ccd17285-bbb4-4dcb-84a8-330f45199afd",
+      "Name": "odAccessKey",
+      "Label": "Access Key (Kind-of Optional)",
+      "HelpText": "An Access Key with permissions to create the desired EC2 instance.\nNote: If empty, this step will attempt to use the value contained in the Machine Environment Variable \"AWS\\_ACCESS\\_KEY\".\n\nFurther Reading:\n[https://docs.aws.amazon.com/general/latest/gr/managing-aws-access-keys.html](https://docs.aws.amazon.com/general/latest/gr/managing-aws-access-keys.html)",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      },
+      "Links": {}
+    },
+    {
+      "Id": "6f058614-92a0-4696-bb56-b6342a8d5466",
+      "Name": "odSecretKey",
+      "Label": "Secret Key (Kind-of Optional)",
+      "HelpText": "The Secret Key associated with the above Access Key.\nNote: If empty, this step will attempt to use the value contained in the Machine Environment Variable \"AWS\\_SECRET\\_KEY\".\n\nFurther Reading:\n[https://docs.aws.amazon.com/general/latest/gr/managing-aws-access-keys.html](https://docs.aws.amazon.com/general/latest/gr/managing-aws-access-keys.html)",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      },
+      "Links": {}
+    }
+  ],
+  "$Meta": {
+    "ExportedAt": "2018-01-30T12:44:07.356Z",
+    "LastModifiedBy": "tclydesdale",
```

Comment:
The `LastModifiedBy` property should be outside of the `$Meta` property (applies to all steps in this PR)